File: Module 2/10/answer.c
#include<stdio.h>
int main() {
    int n;
    scanf("%d",&n);
    int num;
    for(int i=0;i<n;i++)
    {
        scanf("%d",&num);
        int rev=0;
        while(num!=0)
        {
            rev = (rev*10)+num%10;
            num = num/10;
        }
        printf("%d\n",rev);
    }
    return 0;
}

File: Module 3/3/answer.py
num = input()
rev = num[::-1]
num = int("".join(num))
rev = int("".join(rev))
rev_square = str(rev*rev)
rev_rev_square = rev_square[::-1]
rev_rev_square = int("".join(rev_rev_square))
if num*num == rev_rev_square:
    print("ADAM")
else:
    print("NOT ADAM")
